servlet-map/src/main/resources/locale/messages_fi.properties

```diff
@@ -1,61 +1,61 @@
 logout=Kirjaudu ulos
-invalid_password_or_username=Virheellinen käyttäjätunnus tai salasana!!
-username=Käyttäjätunnus
+invalid_password_or_username=Virheellinen k�ytt�j�tunnus tai salasana!!
```

Comment:
There seems to be some line-ending or whitespace differences in this file. @ZakarFin could you check which one's correct

Reply:
It's a character encoding change. I wonder if the swedish version has the same problem